controllers/onboardingController.js

```diff
@@ -108,7 +109,7 @@ module.exports.postForm = async function (req, res) {
     }
 
     function shouldBeOnlyUsername(field, value) {
-      if (!value || (!value.startsWith('http') && !value.startsWith('https') && !value.includes('/'))) {
+      if (isValidGithubUserName(value)) {
         return value;
       }
       formValidationErrors.push(`${field} : la valeur doit être le nom du membre seul et ne doit pas être l'URL du membre`);
```

Comment:
On pourrait ajouter à la fin du message d'erreur "... ni commencer avec '@'"
```suggestion
      formValidationErrors.push(`${field} : la valeur doit être le nom du membre seul et ne doit pas être l'URL du membre ni commencer avec "@"`);
```

Reply:
Même si dans un navigateur correcte on ne verra jamais ce message car la regexp dans l'input ne nous permettra pas de soumettre...